gammapy/visualization/utils.py

```diff
@@ -26,6 +28,16 @@
 }
 
 
+def add_colorbar(img, ax, label=None):
+    """Add colorbar to a given axis."""
+    divider = make_axes_locatable(ax)
+    cax = divider.append_axes(position="right", size="5%", pad="2%")
```

Comment:
One could introduce an argument `axes_loc={}` with the keywords for this method, such that users can modify it. The defaults can best set using `axes_loc.setdefult()`. But keeping the hard-coded values is probably also fine.